pages/admin/users.vue

```diff
@@ -1,13 +1,340 @@
+                </VBtn>
+              </div>
+            </VCard>
+          </template>
+        </VDialog>
+      </div>
+    </div>
+
+    <VDivider class="mb-5 mt-2" />
+
+    <!-- pagination -->
+    <div class="d-flex align-center justify-space-between pagination-wrapper">
+      <VPagination
+        v-model="currentPage"
+        :ripple="false"
+        :length="pages.length"
+        :total-visible="4"
+        density="comfortable"
+      />
+
+      <div
+        class="d-flex align-center"
+        style="text-wrap: nowrap; min-width: fit-content"
+      >
+        <p class="text-subtitle-1 mr-4 font-weight-medium">
+          {{ $t('admin.users.pagesize') }}
+        </p>
+        <VSelect
+          v-model="pageSize"
+          density="compact"
+          variant="outlined"
+          :items="pageSizes"
+          :hide-details="true"
+        />
+      </div>
+    </div>
+
+    <!-- user list -->
+    <VTable hover style="overflow: hidden" class="mb-5">
+      <thead>
+        <tr>
+          <th>{{ $t('admin.users.attributes.select') }}</th>
+          <th>{{ $t('admin.users.attributes.name') }}</th>
+          <th>{{ $t('admin.users.attributes.usertype') }}</th>
+          <th>{{ $t('admin.users.attributes.email') }}</th>
+          <th>{{ $t('admin.users.attributes.year') }}</th>
+          <th>{{ $t('admin.users.attributes.program') }}</th>
+          <th>{{ $t('admin.users.attributes.company') }}</th>
+          <th class="text-center">UID</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(user, index) in pages[currentPage - 1]" :key="index">
+          <!-- select user -->
+          <td>
+            <VCheckbox v-model="selected[index]" :hide-details="true" />
+          </td>
+
+          <!-- static information -->
+          <td>{{ user.name }}</td>
+          <td>{{ user.userType ?? 'basic' }}</td>
+          <td>{{ user.email }}</td>
+          <td>{{ user.currentYear ?? '-' }}</td>
+          <td>{{ user.studyProgram ?? '-' }}</td>
+          <td>
+            {{
+              /* @ts-ignore */
+              companies[user.companyUID]?.name ?? '-'
+            }}
+          </td>
+
+          <!-- copy user id -->
+          <td>
+            <VTooltip location="top" color="primary">
+              <template #activator="{ props }">
+                <VBtn
+                  v-bind="props"
+                  size="small"
+                  variant="text"
+                  icon="mdi-content-copy"
+                  color="primary"
+                  @click="copy(user.uid)"
+                />
+              </template>
+
+              {{
+                copied && text === user.uid
+                  ? $t('admin.users.copied')
+                  : user.uid
+              }}
+            </VTooltip>
+          </td>
+        </tr>
+      </tbody>
+    </VTable>
   </div>
 </template>
+
+<style scoped lang="scss">
+  .filters {
+    gap: 0.5rem;
```

Comment:
Might mostly be my fault (as I am used to using `rem`), but should we try to keep it consistent when it comes to distance values? If so, this should be refactored to `8px`. Not a big deal for me, but what do you think?

Reply:
I agree, we should use rem. I haven't been concise enough with it.